APCAppCore/APCAppCore/UI/Components/APCBadgeLabel.m

```diff
@@ -92,4 +92,10 @@ - (void)setText:(NSString *)text
     [self setNeedsDisplay];
 }
 
+- (void) tintColorDidChange
```

Comment:
@syoung-smallwisdom did these changes and they still work fine.  Good feedback as always!